### DISCUSSION TITLE
November 2, 2021

### DISCUSSION THREAD

**Discussion by manuelalferez** [pmascaraque/iycdangbo#15] [Meetings]
# Meetings Notes:

**Comment by jesi-rgb** [comment]
- Diseñar footer
- Diseñar editor blog
- Diseñar sección noticias
- Diseñar sección donativos

**Comment by manuelalferez** [comment]
- domain too long: `infanciaycirugiaendangbo.org`     ----->    `dangbo.org`

**Comment by manuelalferez** [comment]
Images: 
https://unsplash.com/s/photos/africa

**Comment by manuelalferez** [comment]
Possible technologies for Noticias section:

1. https://draftjs.org
2. https://www.tiny.cloud/docs/demo/basic-example

**Comment by manuelalferez** [comment]
[List of countries and dependencies by number of physicians](https://en.wikipedia.org/wiki/List_of_countries_and_dependencies_by_number_of_physicians)

**Comment by manuelalferez** [comment]
Scrolling and effects: 
- https://openbase.com/categories/js/best-react-scroll-libraries
- https://www.npmjs.com/package/react-scroll
